docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -166,6 +166,13 @@ For the following steps, change into a clean directory (for example a directory
 These are the only steps which aren't idempotent and have to be completed uninterrupted in one go.
 If you have to recreate the install config or any of the generated manifests you need to rerun all of the subsequent steps.
 
+[NOTE]
+--
+You can add options to the `install-config.yaml` as per upstream configuration. Have a look at the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example].
```

Comment:
```suggestion
You can add options to the `install-config.yaml` as per upstream configuration.
Have a look at the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example].
```

New line per sentences as per https://syn.tools/syn/references/style-guide.html#_asciidoc_style

Reply:
I'm having a hard time understanding "as per upstream configuration"...
Doesn't sound like proper English to me. What did you mean? "Similar to...", something else?
Prolly best to reword this sentence

Reply:
I have updated the sentence, please let me know if this is more understandable.